### PR TITLE
Divine sage building a celo based remittance smart contract and interacting with it using python

### PR DESCRIPTION
**Pull request by Divine572**
Thanks for submitting this tutorial for Celo SageBefore you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x]  My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x]  The title of the Pull Request matches the title of my tutorial.
- [x]  I've followed the tutorial template structure.
- [x]  I've proofread the tutorial for spelling and grammar mistakes.
- [x]  I've thoroughly explained the code, with inline code comments and text explanations.
- [x]  I've run a linter & prettifier to format my code blocks.
- [x]  I've run the tutorial to test it and the code works.NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.Reviewers - [@joenyzio](https://github.com/joenyzio) & [@viral-sangani](https://github.com/viral-sangani)

### CODE REVIEW

**Review by Julius170 — COMMENTED** [review]
Hi @Divine572 Ive left some comment here, you can now make the changes

**Review by Julius170 — COMMENTED** [review]
Hi @Divine572 Ive left some comment here, you can now make the changes

**Comment by Divine572** [comment]
@Julius170 please you can check now. I have made the changes

File: blog/04-05-2023-building-a-celo-based-remittance-smart-contract-and-interacting-with-it-using-python/building-a-celo-based-remittance-smart-contract-and-interacting-with-it-using-python.md [hunk 1]
```diff
@@ -0,0 +1,459 @@
...
+On your terminal, run the following command to deploy your smart contract:
+
+```bash
+python deploy.py
+```
+
+![images](https://user-images.githubusercontent.com/69091491/236292089-5e6dbce9-381b-4beb-871d-39ab2ad47563.png)
+
+
+### Step 4: In
```
[the remark this answers is not in the document]
Comment (Julius170) [reply blog/04-05-2023-building-a-celo-based-remittance-smart-contract-and-interacting-with-it-using-python/building-a-celo-based-remittance-smart-contract-and-interacting-with-it-using-python.md:332]: Still haven't corrected the error here

**Review by Julius170 — COMMENTED** [review]
Reference to previous comments

**Review by Julius170 — COMMENTED** [review]
Reference to previous comments

**Review by Julius170 — COMMENTED** [review]
Reference to previous comments

File: blog/04-05-2023-building-a-celo-based-remittance-smart-contract-and-interacting-with-it-using-python/building-a-celo-based-remittance-smart-contract-and-interacting-with-it-using-python.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Divine572) [reply blog/04-05-2023-building-a-celo-based-remittance-smart-contract-and-interacting-with-it-using-python/building-a-celo-based-remittance-smart-contract-and-interacting-with-it-using-python.md]: I have made the changes

**Comment by viral-sangani** [comment]
Closing Celo Sage PRs

**Pull request closed without merging**